pixart.js

```diff
@@ -1 +1,30 @@
+var square = $('<div class="square"></div>');
 
+for (var i=0; i<8000; i++){
+  $('body').append(square.clone());
+  // console.log('so square broh');
+}
+
+$(document).on("ready", function() {
+  // console.log('jquery ready broh');
+
+  $('button').on("click", getColor);
+  $('.square').on("mouseover", changeSquare);
+
+})
+
+function getColor(evt) {
+  // console.log('button clicked broh');
+  evt.preventDefault();
+  var brushColor = $('#color-field').val();
+  // console.log(colorName);
+  $('.brush').css("background-color", brushColor);
+  // console.log($('.brush').css("background-color"));
+}
+
+function changeSquare() {
+  // console.log(this);
+  var squareColor = $('#color-field').val();
+  $(this).css("background-color", squareColor);
```

Comment:
instead of grabbing from the input, maybe just use the color that the brush box is, otherwise, what the point in setting it in the above event?